### ISSUE TITLE
Navbar is transparent

### ISSUE THREAD

**Issue by kaxline** [Methodician/guytime#79]

**Comment by bsteinbeigle** [comment]
Thank you for noticing this! So is the top menu. Its more of an annoyance for now. Would this just involve some CSS tweaking?

**Comment by kaxline** [comment]
@bsteinbeigle Yeah, I already fixed this while I was learning the code base. Should go out with next deploy.

**Comment by bsteinbeigle** [comment]
@kaxline lets close this issue.
